spec/system/vendor_api/vendor_receives_application_spec.rb

```diff
@@ -71,10 +71,11 @@ def then_it_should_include_the_data_from_the_application_form
         qualifications: {
           gcses: [
             {
+
               id: @application.english_gcse.id,
```

Comment:
This is returning nil in the test which doesn't seem correct. I need to speak to @raamSoftwire and @duncanjbrown to confirm

Reply:
I spoke to @duncanjbrown about this yesterday and I'm currently working on tidying this up.

For the API we want to return structured GCSEs as separate qualifications and so we split them out. But this means that the separated GCSEs don't have IDs (because they aren't stored in the DB). We decided to copy the ID from the original structured GCSE and to add a warning to the API that the IDs may not be unique.

Reply:
Great. That makes complete sense 👍